agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -1593,49 +1593,6 @@ bool slave_thread::handle_cmdu_platform_manager_message(
         }
         break;
     }
-    case beerocks_message::ACTION_PLATFORM_WIFI_CONFIGURATION_UPDATE_REQUEST: {
-        auto response =
-            cmdu_rx
-                .addClass<beerocks_message::cACTION_PLATFORM_WIFI_CONFIGURATION_UPDATE_REQUEST>();
-        if (response == nullptr) {
-            LOG(ERROR) << "addClass cACTION_PLATFORM_WIFI_CONFIGURATION_UPDATE_REQUEST failed";
-            return false;
-        }
-        LOG(INFO) << "ACTION_PLATFORM_WIFI_CONFIGURATION_UPDATE_REQUEST config_start="
-                  << int(response->config_start());
-
-        if (slave_state != STATE_OPERATIONAL &&
-            slave_state != STATE_WAIT_FOR_WIFI_CONFIGURATION_UPDATE_COMPLETE &&
-            slave_state != STATE_WAIT_FOR_ANOTHER_WIFI_CONFIGURATION_UPDATE) {
-            LOG(DEBUG) << "invalid slave state - ignore wifi configuration notification";
-        } else if (!response->config_start()) {
-            LOG(DEBUG) << "WIFI_CONFIGURATION_UPDATE_COMPLETE";
-            if (detach_on_conf_change) {
-                LOG(DEBUG) << "detach occurred on wifi conf change, slave reset!";
-                slave_reset();
-            } else if (
-                master_socket) { // if backhaul disconnects before we get WIFI_CONFIGURATION_UPDATE_COMPLETE, so the slave will continue on its current state
-                LOG(DEBUG) << "WIFI_CONFIGURATION_UPDATE_COMPLETE! goto STATE_OPERATIONAL";
-                slave_state = STATE_OPERATIONAL;
-            }
-
-        } else if (slave_state == STATE_WAIT_FOR_WIFI_CONFIGURATION_UPDATE_COMPLETE) {
-            // response->config_start == 1, but prev conf change is not finished yet
-            slave_state_timer =
-                std::chrono::steady_clock::now() +
-                std::chrono::seconds(beerocks::SON_SLAVE_WAIT_AFTER_WIFI_CONFIG_UPDATE_SEC);
```

Comment:
SON_SLAVE_WAIT_AFTER_WIFI_CONFIG_UPDATE_SEC is no longer used so should be removed.

Reply:
removed.